magellan-legacy/build.gradle

```diff
@@ -0,0 +1,44 @@
+apply plugin: 'com.android.library'
+
+group = GROUP
+version = VERSION_NAME
+
+android {
+  compileSdkVersion 30
+  buildToolsVersion BUILD_TOOLS_VERSION
+
+  resourcePrefix 'magellan_'
+
+  defaultConfig {
+    minSdkVersion 15
+    targetSdkVersion 30
+    versionCode 1
+    versionName "1.0"
```

Comment:
nit: maybe we should be using variables here?

Reply:
Yeah. We should. Since we seem to have a lot of modules, im gonna migrate to the buildSrc pattern here too. But in a separate PR.